pkg/target/target.go

```diff
@@ -67,9 +67,9 @@ var _ Target = &targetImpl{}
 
 // NewTarget returns a new target. This function does not perform any validation,
 // so the returned target can be invalid.
-func NewTarget(gardenName, projectName, seedName, shootName string) Target {
+func NewTarget(identity, projectName, seedName, shootName string) Target {
 	return &targetImpl{
-		Garden:  gardenName,
+		Garden:  identity,
```

Comment:
I am not so happy with the renaming from `gardenName` to `identity`. I think we should have better not changed this and only use the cluster identity as name. What do you think?